server/src/helpers/character/StealHelper.ts

```diff
@@ -92,12 +92,13 @@ export class StealHelper extends BaseService {
       const handName = this.game.characterHelper.getEmptyHand(char);
       if (!handName) return;
 
-      const item = sample(target.items.sack.items);
-      this.game.inventoryHelper.removeItemFromSack(target, item);
+      const item = sample(target.items.sack.items) as ISimpleItem;
```

Comment:
can you add an if(!item) return this.sendMessage('you could not find anything to take!')

Reply:
It does this above that
else if (target.items.sack.items.length > 0)

Reply:
It does a check at the very top. That being said this method has other issues that should be fixed in a new PR.